simGanESS/source/geometries/GaP.cc

```diff
@@ -245,31 +282,50 @@ void GaP1::DefineConfigurationParameters()
   msg_->DeclarePropertyWithUnit("specific_vertex", "mm",  specific_vertex_, "Set generation vertex.");
 }
 
-void GaP1::BuildTPC(G4Material* gas, G4Material* cath_mat, G4Material* gate_mat, G4Material* anode_mat, G4LogicalVolume* logic_vessel)
+void GaP::BuildTPC(G4Material* gas, G4Material* cath_mat, G4Material* gate_mat, G4Material* anode_mat, G4LogicalVolume* logic_vessel_steel)
 {
     //Gas
     G4double drift_length_  = 19.825*mm;
     G4double el_length_     = 10.775*mm;
     G4double pmt_gap_       = 23.2  *mm;
     G4double buffer_        = 5     *mm; // Cap to cathode buffer
 
+    G4Tubs          *solid_vessel = new G4Tubs("GasVessel", 0, vessel_rad_, vessel_length_/2 , 0., 360.*deg);
+    G4LogicalVolume *logic_vessel = new G4LogicalVolume(solid_vessel, gas_, "GasVessel");
```

Comment:
Use gas instead of gas_ (mistake on my side, but since you are at it, fixed it ;) )